.gitattributes

```diff
@@ -0,0 +1,2 @@
+*.sh   eol=lf
+*.bat  eol=crlf
```

Comment:
If `bat` is added, `cmd` should be too. The repo has some `cmd` files. (not for production, only for testing)